### PR TITLE
Adding "openssh" package to bazel layers

### PR DESCRIPTION
**Pull request by strican**
When building a Bazel project with dependencies on private repositories using the Bazel docker image, builds currently fail with the `error: cannot run ssh: No such file or directory`. Adding `ssh` to the bazel layer will make sure it is included in the image.

### CODE REVIEW

**Comment by nlopezgi** [comment]
hi @strican 
Thanks for opening this PR.
Which rules exactly are causing this failure? Is this just for repo rules that need to access a private repo or something else? Need to clarify to understand if this is a core requirement of Bazel or if its needed strictly for some subset of rules.

**Comment by strican** [comment]
There was a discussion about this issue in the Bazel slack group. I was trying to use the image as part of a Buildkite CI task and was just calling `bazel build ...`. I believe it failed at `git_repository`. Here's a slightly sanitized output of the build failure:

```
- /workdir/WORKSPACE:63:1
--
  | ERROR: /workdir/X/BUILD.bazel:12:1: no such package '@X//...': Traceback (most recent call last):
  | File "/root/.cache/bazel/_bazel_root/ec321eb2cc2d0f8f91b676b6d4c66c29/external/bazel_tools/tools/build_defs/repo/git.bzl", line 234
  | _clone_or_update(ctx)
  | File "/root/.cache/bazel/_bazel_root/ec321eb2cc2d0f8f91b676b6d4c66c29/external/bazel_tools/tools/build_defs/repo/git.bzl", line 74, in _clone_or_update
  | fail(("error cloning %s:\n%s" % (ctx....)))
  | error cloning X:
  | + cd /root/.cache/bazel/_bazel_root/ec321eb2cc2d0f8f91b676b6d4c66c29/external
  | + rm -rf /root/.cache/bazel/_bazel_root/ec321eb2cc2d0f8f91b676b6d4c66c29/external/X /root/.cache/bazel/_bazel_root/ec321eb2cc2d0f8f91b676b6d4c66c29/external/X
  | + git clone '--shallow-since=1556288237 -0400' git@github.com:improbable/X.git /root/.cache/bazel/_bazel_root/ec321eb2cc2d0f8f91b676b6d4c66c29/external/X
  | error: unknown option `shallow-since=1556288237 -0400'
  | usage: git clone [<options>] [--] <repo> [<dir>]
  |  
  | -v, --verbose         be more verbose
  | -q, --quiet           be more quiet
  | --progress            force progress reporting
  | -n, --no-checkout     don't create a checkout
  | --bare                create a bare repository
  | --mirror              create a mirror repository (implies bare)
  | -l, --local           to clone from a local repository
  | --no-hardlinks        don't use local hardlinks, always copy
  | -s, --shared          setup as shared repository
  | --recursive           initialize submodules in the clone
  | --recurse-submodules  initialize submodules in the clone
  | --template <template-directory>
  | directory from which templates will be used
  | --reference <repo>    reference repository
  | --dissociate          use --reference only while cloning
  | -o, --origin <name>   use <name> instead of 'origin' to track upstream
  | -b, --branch <branch>
  | checkout <branch> instead of the remote's HEAD
  | -u, --upload-pack <path>
  | path to git-upload-pack on the remote
  | --depth <depth>       create a shallow clone of that depth
  | --single-branch       clone only one branch, HEAD or --branch
  | --separate-git-dir <gitdir>
  | separate git dir from working tree
  | -c, --config <key=value>
  | set config inside the new repository
  |  
  | + git clone git@github.com:improbable/X.git /root/.cache/bazel/_bazel_root/ec321eb2cc2d0f8f91b676b6d4c66c29/external/X
  | Cloning into '/root/.cache/bazel/_bazel_root/ec321eb2cc2d0f8f91b676b6d4c66c29/external/X'...
  | error: cannot run ssh: No such file or directory
  | fatal: unable to fork
```

**Comment by strican** [comment]
Also note that as @tmc pointed out in Slack, the [details page](https://console.cloud.google.com/marketplace/details/google/bazel?pli=1) for the image seems to indicate that OpenSSH was intended to be there.

File: layers/ubuntu1404/bazel/BUILD [hunk 1]
```diff
@@ -38,6 +38,7 @@ download_pkgs(
         "binutils",
         "file",
         "git",
+        "openssh",
```
Comment (nlopezgi) [inline layers/ubuntu1404/bazel/BUILD]: can we use openssh-client instead?

Comment (strican) [reply layers/ubuntu1404/bazel/BUILD]: Thanks for catching that. Fixed.

**Pull request merged**